components/GroupInvite.tsx

```diff
@@ -0,0 +1,37 @@
+import { ICheck, IClose } from './icons';
+
+type Props = {
+  groupName: string;
+  instrument: string;
+  locations: string[];
+  description: string;
+};
+
+export default function GroupInvite({
```

Comment:
Rename to `ProfileGroupInvite`, move it to profile folder, `export function GroupInvite` and not default. export default at the bottom of the file. Check other files. Then export in `index.ts`

- [ ] rename file
- [ ] move file
- [ ] export in index.ts